test/dotnet/Integration/IntegrationTestHelpersCopy.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.Azure.Cosmos;
```

Comment:
Any reason these files are called `Copy`?

Reply:
Because I added tests directly on the Microsoft.Azure.WebJobs.Extensions.Redis.Tests and didn't want to override what they had

Reply:
These show up as completely different files, did you change the location?

Reply:
I'm not sure. I originally committed them on another branch and then merged it here

Reply:
If you're talking about the directory, I made the other files as their own projects in the same Microsoft.Azure.WebJobs.Extensions.Redis.Test rather than directly inside

Reply:
did you create these files from scratch or did you only add some tests to it? In the latter case, these tests should show up as additions instead of new files. I do think new files make sense, so maybe you can rename them and remove any tests you did not write.

Reply:
I changed one of the tests they had which is why I made another file